packages/coil-extension/src/background/services/BackgroundScript.ts

```diff
@@ -724,36 +734,50 @@ export class BackgroundScript {
     }
 
     const receiver = stream.getPaymentPointer()
-    const { assetCode, assetScale, exchangeRate } = stream.getAssetDetails()
```

Comment:
See: https://github.com/coilhq/web-monetization-projects/pull/567#discussion_r419924429

Removed as the new tipping endpoint will return the exact amount sent